packages/ui/src/proposals/modals/AddNewProposal/helpers.ts

```diff
@@ -402,5 +409,8 @@ export const schemaFactory = (api?: Api) => {
         .required(),
       channelCashoutsEnabled: Yup.boolean(),
     }),
+    updatePalletFrozenStatus: Yup.object().shape({
+      frozen: Yup.boolean().test(equalValueContext('palletFrozenStatus', 'execution')).required('Field is required'),
```

Comment:
Also this should test whether the value is different from the context not equal to it. So I think this function should be renamed:
```suggestion
      frozen: Yup.boolean().test(differentFromContext('palletFrozenStatus', 'execution')).required('Field is required'),
```